.github/workflows/examples.yml

```diff
@@ -0,0 +1,61 @@
+name: Build Save-API CLI examples
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  build_and_test_with_code_coverage:
+    name: Build, test and upload code coverage
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          # required for correct codecov upload
+          fetch-depth: 0
+          submodules: true
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: 11
+          distribution: zulu
+      - name: Install system packages
+        # libcurl is needed for ktor-client-curl, libc-bin for orchestrator
+        run: sudo apt-get update && sudo apt-get install -y libcurl4-openssl-dev libc-bin
+      - name: Cache konan
+        uses: actions/cache@v3
+        with:
+          path: ~/.konan
+          key: ${{ runner.os }}-gradle-konan
+          restore-keys: |
+            ${{ runner.os }}-gradle-konan
+
+      - uses: burrunan/gradle-cache-action@v1
+        with:
+          gradle-version: wrapper
+          # additional files to calculate key for dependency cache
+          gradle-dependencies-cache-key: |
+            gradle/libs.versions.toml
+          arguments: |
+            save-api:build
+            -DincludeSaveApi='true'
+            -x detekt
+          properties: |
+            org.gradle.caching=true
+            detekt.multiplatform.disabled=true
+
+      - name: Upload gradle reports
+        if: ${{ always() }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: gradle-reports
+          path: '**/build/reports/'
+          retention-days: 1
+
+      - name: Code coverage report
```

Comment:
codecov is needed on the main workflow only

Reply:
Removed